### PR TITLE
feat/tech-51

### PR DESCRIPTION
**Pull request by letbueno**
# Description
- transfer variables to constants

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] It is urgent
- [ ] <span style="color:#d44037">Dangerous change</span>

### CODE REVIEW

File: config/env.yml [hunk 1]
```diff
@@ -2,35 +2,35 @@ default: &default
...
     url: <%= ENV['REDIS_URL'] %>
   blockchain:
-    scan_url: <%= ENV['BLOCKCHAIN_SCAN_URL'] %>
+    scan_url: "https://mumbai.polygonscan.com/tx/"
   the_graph:
-    url: <%= ENV['THE_GRAPH_URL'] %>
+    url: "https://api.thegraph.com/subgraphs/name/ribondao/subgraphribon"
   integration_address:
-    base_url: <%=  ENV['INTEGRATION_ADDRESS_BASE_URL']  %>
+    base_url: "https://dapp.ribon.io/integration/"
```
Comment (NickNish09) [inline config/env.yml:9]: maybe here the address would be
`https://dapp.ribon.io/?integration_id=`

Comment (letbueno) [reply config/env.yml:9]: i really don't know because in the tests is integration/:id

Comment (NickNish09) [reply config/env.yml:9]: oh well, I think the tests are wrong then. This should be something like this route on the frontend:
[https://dapp.ribon.io/?integration_id=e76b2de0-3357-4d6e-b2ce-1ea091347f46](https://dapp.ribon.io/?integration_id=e76b2de0-3357-4d6e-b2ce-1ea091347f46)

If we go to this route: https://dapp.ribon.io/integration/ there's nothing. We could change the frontend to use this, but I think that the idea is to maintain the causes page as the root page

Comment (letbueno) [reply config/env.yml:9]: yesss, i will change in tests

**Review by NickNish09 — APPROVED** [review]

**Pull request merged**